### ISSUE TITLE
Crash when playing video

### ISSUE THREAD

**Issue by bmarty** [vector-im/element-android#3179]
Due to recent change on coroutines

```
android.view.ViewRootImpl$CalledFromWrongThreadException: 
  at android.view.ViewRootImpl.checkThread (ViewRootImpl.java:10744)
  at android.view.ViewRootImpl.requestLayout (ViewRootImpl.java:2209)
  at androidx.recyclerview.widget.RecyclerView.requestLayout (RecyclerView.java:2)
  at android.view.View.requestLayout (View.java:27055)
  at android.widget.RelativeLayout.requestLayout (RelativeLayout.java:380)
  at android.view.View.requestLayout (View.java:27055)
  at android.view.View.setFlags (View.java:17525)
  at android.view.View.setVisibility (View.java:12466)
  at android.widget.ImageView.setVisibility (ImageView.java:1702)
  at im.vector.lib.attachmentviewer.DefaultVideoLoaderTarget.arrangeForVideoReady (VideoLoaderTarget.kt:4)
  at im.vector.lib.attachmentviewer.DefaultVideoLoaderTarget.onVideoFileReady (VideoLoaderTarget.kt:4)
  at im.vector.app.features.media.BaseAttachmentProvider$loadVideo$2.invokeSuspend (BaseAttachmentProvider.kt:16)
  at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith (ContinuationImpl.kt:3)
  at kotlinx.coroutines.DispatchedTask.run (DispatchedTask.kt:15)
  at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely (CoroutineScheduler.kt:1)
  at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run (CoroutineScheduler.kt:10)
```

**Comment by Dominaezzz** [comment]
I'm so sorry, this is my bad.
https://github.com/vector-im/element-android/blob/0e71dfa8e105062e8ff5eb50b5d6cc4baa7f7ca5/vector/src/main/java/im/vector/app/features/media/BaseAttachmentProvider.kt#L156
This line should be `GlobalScope.launch(Dispatches.Main)`.

**Comment by bmarty** [comment]
Thanks @Dominaezzz, what about the other usage of `GlobalScope.launch { ... }` added in b6f4be289419017cb3b122c5f0e8906119911229 ?

**Comment by Dominaezzz** [comment]
Ah I just looked at the specific one that caused the crash.

I think all of them should be changed to `GlobalScope.launch(Dispatches.Main)`, at least it would maintain previous (correct) behaviour.

(Please mention me in future coroutine crashes like this, I only found this issue by chance)

**Comment by bmarty** [comment]
Fixed in Element Android 1.1.5
